From 7274b1efb99ecb8daa67302ade84e6c0b6d337c0 Mon Sep 17 00:00:00 2001
From: Neil Armstrong <narmstrong@baylibre.com>
Date: Thu, 25 Feb 2021 09:44:37 +0100
Subject: [PATCH 13/40] FROMGIT: arm: meson: remove static ethernet memory
 power domain enable

The ethernet memory power domain is handled by the meson-ee-pwrc driver,
delete the static code.

Signed-off-by: Neil Armstrong <narmstrong@baylibre.com>
---
 arch/arm/mach-meson/board-g12a.c | 3 ---
 arch/arm/mach-meson/board-gx.c   | 3 ---
 2 files changed, 6 deletions(-)

diff --git a/arch/arm/mach-meson/board-g12a.c b/arch/arm/mach-meson/board-g12a.c
index bfd40fe14f..cc7e01d014 100644
--- a/arch/arm/mach-meson/board-g12a.c
+++ b/arch/arm/mach-meson/board-g12a.c
@@ -126,9 +126,6 @@ void meson_eth_init(phy_interface_t mode, unsigned int flags)
 		printf("Invalid Ethernet interface mode\n");
 		return;
 	}
-
-	/* Enable power gate */
-	clrbits_le32(G12A_MEM_PD_REG_0, G12A_MEM_PD_REG_0_ETH_MASK);
 }
 
 #if CONFIG_IS_ENABLED(USB_DWC3_MESON_G12A) && \
diff --git a/arch/arm/mach-meson/board-gx.c b/arch/arm/mach-meson/board-gx.c
index 876405f0d4..cae7af5afb 100644
--- a/arch/arm/mach-meson/board-gx.c
+++ b/arch/arm/mach-meson/board-gx.c
@@ -142,9 +142,6 @@ void meson_eth_init(phy_interface_t mode, unsigned int flags)
 		printf("Invalid Ethernet interface mode\n");
 		return;
 	}
-
-	/* Enable power gate */
-	clrbits_le32(GX_MEM_PD_REG_0, GX_MEM_PD_REG_0_ETH_MASK);
 }
 
 #if CONFIG_IS_ENABLED(USB_DWC3_MESON_GXL) && \
-- 
2.17.1

